<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSum">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSubtract">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEquals">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDivide">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMultiply">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonComma">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>256</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="historyLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Operation History</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
